InvenTree/plugin/serializers.py

```diff
@@ -171,6 +171,26 @@ def save(self):
 
 class PluginConfigEmptySerializer(serializers.Serializer):
     """Serializer for a PluginConfig."""
+    ...
+
+
+class PluginActivateSerializer(serializers.Serializer):
+    """Serializer for activating a plugin"""
```

Comment:
Question: Should this maybe be renamed to reflect that it can activate/deactivate a plugin?

Reply:
Done :)